joomla/joomla-cms
# [4.0] [passwordstrength.js] Migrate es5 code to es6

### Summary of Changes

Migrated passwordstrength.js to es6

### Testing Instructions
run npm install
test Joomla core functionality like
* javascript translations
* check error messages creation in forms / ajax forms
* check webcomponents loaded correctly
* when registering a new user or adding new user in backend, check if the password strength criteria matches Password options set in Users: Password Options.

### Expected result
Passwordstrength should work same as before


### Actual result

Passwordstrength works same as before
